src/pages/[locale]/media/index.tsx

```diff
@@ -0,0 +1,22 @@
+import {GetServerSideProps} from 'next';
```

Comment:
We no longer have the `pages/[locale]` folder. Need to move `media` directly into `pages`.